src/store/WalletStore.ts

```diff
@@ -32,7 +33,15 @@ export default defineStore("WalletStore", {
 			this.walletKind = walletType;
 		},
 		setWebMode(webMode: WebModeTypes) {
-			this.webMode = webMode;
+			// my algo wallet has issue with ref in vue
+			// for reference : https://github.com/randlabs/myalgo-connect/issues/71
+			let myalgo = shallowRef() as any;
+			if (this.walletKind === WalletType.MY_ALGO) {
+				myalgo = webMode;
```

Comment:
You have not set webmode for this?? How are you planning to use webmode for this wallet?

Reply:
in the next line I m setting the webmode to myalgo, actually the error is because of ref in vue, which is fixed by this. my algo is also a webmode